docs/data/rpc/api-reference/structure/data-format.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 60
+title: Data Format
+---
+
+import { ExampleResponse } from "@site/src/components/ExampleResponse";
+import { AttributeTable } from "@site/src/components/AttributeTable";
+
+### XDR Format
+
+In the Stellar network, transactions are encoded using a standardized protocol called [External Data Representation](https://en.wikipedia.org/wiki/External_Data_Representation) (XDR).
```

Comment:
This should cross-ref https://developers.stellar.org/docs/learn/encyclopedia/data-format/xdr, instead.